hack/verify-kube-version.sh

```diff
@@ -8,4 +8,10 @@ KUBECTL_GIT_VERSION="${KUBE_GIT_VERSION//v1./}"
 if [[ "${KUBECTL_GO_MOD_VERSION}" != "${KUBECTL_GIT_VERSION}" ]]; then
   os::log::warning "kubernetes version and kubectl version in go.mod must be equal, please update KUBE_GIT_VERSION"
   exit 1
-fi
\ No newline at end of file
+fi
+
+KUBECTL_DOCKER_VERSION=$(grep "kubectl=" images/cli/Dockerfile.rhel | sed 's/      io.openshift.build.versions="kubectl=1.//' | sed 's/" \\//')
+if [[ "${KUBECTL_GO_MOD_VERSION}" != "${KUBECTL_DOCKER_VERSION}" ]]; then
+  os::log::warning "kubernetes version and kubectl version in images/cli/Dockerfile.rhel must be equal, please update Dockerfile"
```

Comment:
nit: this could be slightly easier for folks to do if you say:

```sh
os::log::warning "kubernetes version and kubectl version in images/cli/Dockerfile.rhel must be equal, please update Dockerfile to set io.openshift.build.versions=\"kubectl=${KUBECTL_GO_MOD_VERSION}\""
```

or something.